Scripts/script-CommonServer.yml

```diff
@@ -817,9 +832,9 @@ script: |-
 
   /**
    * Convert a given object to md list (-) while each value is converted to simple str
```

Comment:
* Convert a given object to md list (-) while each value is converted to a simple string